### DISCUSSION TITLE
Multi table

### DISCUSSION THREAD

**Discussion by pritishbhakat** [gretelai/gretel-synthetics#161] [Q&A]
What is the limitation of multi-table synthetic data in case of relationship an depth of the database

**Comment by amysteier** [comment]
We've tested databased with a depth of 4, and our implementation is designed to handle higher depths.  We'd love to hear if you encounter a limit when synthesizing you databases.  We're also currently testing a new subsetting feature that hasn't been formally released yet. That feature will let you specify some percentage of the database you'd like to extract and train on. You can also specify which tables you want to synthesize and cap the depth you're using this way.
